Library/Homebrew/formulary.rb

```diff
@@ -261,6 +263,12 @@ def self.convert_to_string_or_symbol(string)
     string
   end
 
+  def self.convert_to_deprecate_disable_reason_string_or_symbol(string)
```

Comment:
Ideally these would be `private_class_method` but then it's not accessible inside `Class.new` so probably a bit messy to do.

Reply:
I think it's okay, especially since this whole class is supposed to be a private API

Reply:
It's fine really. Just a habit I'm doing more often but it's probably not worth it.